src/edfi-paging-test/edfi_paging_test/performance_tester.py

```diff
@@ -56,14 +60,42 @@ def fetch_resource(request_client: RequestClient, target_resource: str) -> None:
         )
 
 
+def invalid_resources(
+    openapi_resources: List[str], resources_to_check: List[str]
+) -> List[str]:
+    return [r for r in resources_to_check if r.lower() not in openapi_resources]
+
+
 async def run(args: MainArguments) -> None:
 
     try:
         logger.info("Starting paging volume test...")
         start = time.time()
+
+        openapi_resources: List[str] = get_resources_from_openapi(args.baseUrl)
+        normalized_resources: List[str] = list(
+            map(
+                lambda r: r.removeprefix("/").removeprefix("ed-fi/").lower(),
```

Comment:
Is this so that you can keep my code that adds `ed-fi` if no extension name is provided?

Reply:
Yes, it's to match up with that behavior. I assumed we didn't want to start requiring "edfi/" prefixing when we validate a resource list from the env. And also that casing wasn't important.